spec/system/task_spec.rb

```diff
@@ -1,36 +1,33 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { create(:project) }
+  let(:task)    { create(:task, project_id: project.id) }
```

Comment:
Factoryに`association`をつけると`project_id`を記載しなくて済みますね。
```suggestion
  let(:task)    { create(:task) }
```

Reply:
2821bf1bb7d91e16ee1c8c395f701de96a974729 `factory`に`association`を追加し、`spec/system/task_spec.rb`をリファクタリングしました。
併せて`pending`が出ていた、`spec/models/task_spec.rb`にて`pending`がでないようコメントアウトしました。